tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/timeline/timeline_component.ng.html

```diff
@@ -45,7 +45,10 @@
       >
         <div
           class="execution-digest"
-          [ngClass]="[displayDigest.is_graph ? 'func-graph-execution' : '', i === focusedExecutionDisplayIndex ? 'focused' : '']"
+          [ngClass]="[
+            displayDigest.is_graph ? 'func-graph-execution' : '',
```

Comment:
```
{
  'func-graph-execution': displayDigest.is_graph,
  ...,
  [displayFocusedAlertTypes[i]]: displayFocusedAlertTypes[i] !== null,
}
```

Reply:
This doesn't quite work for me. It would be slightly cleaner if it worked, but what I have right now isn't too bad either.